sympy/polys/tests/test_numberfields.py

```diff
@@ -637,9 +637,9 @@ def test_AlgebraicNumber():
 
     a = AlgebraicNumber(sqrt(2))
     b = to_number_field(sqrt(2))
-    assert a.args == b.args == (sqrt(2), Tuple())
+    assert a.args == b.args == (sqrt(2), Tuple(1, 0))
```

Comment:
I'm ok with fixing it here if you need it for printing to work.  But in a different commit.

If people would really expect `Tuple()` to give the zero element (as you suggested), you could add test?  (although why is that different from `(0,0)`?  convention?)